instrumentation-tests/src/androidTest/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <manifest xmlns:tools="http://schemas.android.com/tools"
     xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.mapbox.navigation.instrumentation_tests.test">
+    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
```

Comment:
Do you still need it? If yes, should it be here or in the main manifest? Is it present in the apk?

Reply:
No, removed. Please note that I removed it from the main manifest after the testing as well to avoid situations when other developers think that we need that permission for something